staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -611,6 +613,7 @@ func CopyConfig(config *Config) *Config {
 			Groups:   config.Impersonate.Groups,
 			Extra:    config.Impersonate.Extra,
 			UserName: config.Impersonate.UserName,
+			UID:      config.Impersonate.UID,
```

Comment:
nit: please sort these fields in the order they appear in the struct (after addressing the first comment).